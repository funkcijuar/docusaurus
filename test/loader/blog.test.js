const path = require('path');
const fs = require('fs');
const loadBlog = require('../../lib/loader/blog');

describe('loadBlog', () => {
  const simpleDir = path.join(__dirname, '__fixtures__', 'simple');
  const customDir = path.join(__dirname, '__fixtures__', 'custom');

  test('simple', async () => {
    const blogDatas = await loadBlog(simpleDir);
    expect(blogDatas).toMatchSnapshot();
  });

  test('custom', async () => {
    const blogDatas = await loadBlog(customDir);
    expect(blogDatas).toMatchSnapshot();
  });
});
