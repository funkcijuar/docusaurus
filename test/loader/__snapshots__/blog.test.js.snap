// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`loadBlog custom 1`] = `
Array [
  Object {
    "content": "

Hello World
",
    "date": 2018-07-28T00:00:00.000Z,
    "path": "/hello.html",
    "title": "Hello, World !",
  },
  Object {
    "content": "

Lorem ipsum
",
    "date": 2018-06-20T00:00:00.000Z,
    "path": "/foo/bar.html",
    "title": "Lorem ipsum",
  },
  Object {
    "content": "
Life is so good
",
    "date": 2018-05-20T00:00:00.000Z,
    "path": "/foo/baz.html",
    "title": "Baz",
  },
]
`;

exports[`loadBlog simple 1`] = `
Array [
  Object {
    "content": "

Hello World
",
    "date": 2018-07-28T00:00:00.000Z,
    "path": "/hello.html",
    "title": "Hello, World !",
  },
]
`;
